src/js/components/DataTable/stories/datatable.stories.js

```diff
@@ -135,6 +135,20 @@ const SimpleDataTable = () => (
   </Grommet>
 );
 
+const ClickableDataTable = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large">
+      {/* eslint-disable no-alert */}
+      <DataTable
+        columns={columns}
+        data={DATA}
+        step={10}
+        onClickRow={event => alert(JSON.stringify(event.datum, null, 2))}
```

Comment:
Since the rows are clickable, I'd expect to be able to move between the rows using keyboard navigation.

Reply:
You should be able to via up and down arrows.

Reply:
Yep that works. sorry I missed that.